client/src/lib/solana/transaction.ts

```diff
@@ -39,19 +39,32 @@ export class SolidTransaction {
     return SolidTransaction.getSolid(connection, recordKey);
   }
 
+  /**
+   * Create and send an instruction to deactivate the DID
+   * @param connection A connection to the blockchain
+   * @param payer The payer of the transaction - this account also receives the lamports stored
+   * @param recordKey
+   * @param authority
+   */
   static async deactivateSolid(
     connection: Connection,
     payer: Account,
-    recordKey: PublicKey
+    recordKey: PublicKey,
+    authority: Account = payer
```

Comment:
You could even separate out the receiver of the drained lamports for maximum flexibility!

Reply:
Yeah I think I won't do that right now to keep the API simple but good to know it's possible.